.github/workflows/build.yml

```diff
@@ -49,8 +58,15 @@ jobs:
   build-stellar-horizon:
     runs-on: ubuntu-latest
     steps:
-    - uses: docker/setup-buildx-action@5146db6c4d81fbfd508899f851bbb3883a96ff9f
-    - name: Build Stellar-Horizon Image
+    - id: cache
+      uses: actions/cache@v3
+      with:
+        path: /tmp/image
+        key: image-stellar-horizon-${{ env.CORE_REPO_BRANCH }}
```

Comment:
confirming on similar usage, was `env.GO_REPO_BRANCH` intended to be serialized into the cache key value?

Reply:
Thakns, this is a typo. Fixed in cffe0da.